integration-tests/opentelemetry-jaeger-remote/src/test/java/io/quarkus/it/opentelemetry/jaeger/OTelJaegerRemoteTest.java

```diff
@@ -0,0 +1,127 @@
+            ExposedPort collectorPort = ExposedPort.tcp(COLLECTOR_PORT);
+            ExposedPort hostPort = ExposedPort.tcp(HEALTH_PORT);
+            Ports portBindings = new Ports();
+            portBindings.bind(queryPort, Ports.Binding.bindPort(QUERY_PORT));
+            portBindings.bind(collectorPort, Ports.Binding.bindPort(COLLECTOR_PORT));
+            portBindings.bind(hostPort, Ports.Binding.bindPort(HEALTH_PORT));
+            CreateContainerResponse container = dockerClient
+                    .createContainerCmd("ghcr.io/open-telemetry/opentelemetry-java/jaeger:1.32")
+                    .withExposedPorts(queryPort, collectorPort, hostPort)
+                    .withHostConfig(newHostConfig().withPortBindings(portBindings))
+                    .withName("jaeger")
+                    .exec();
+            dockerClient.startContainerCmd(container.getId()).exec();
+        }
+    }
+
+    @AfterAll
+    static void teardown() {
+        dockerClient.listContainersCmd().exec()
+                .forEach(container -> {
+                    dockerClient.stopContainerCmd(container.getId()).exec();
+                    dockerClient.removeContainerCmd(container.getId()).exec();
+                });
+    }
+
+    @Test
+    void testJaegerRemoteIntegration() {
+        createTestSpan(openTelemetry);
+        JsonPath spanData = Awaitility.await()
+                .atMost(Duration.ofSeconds(30))
+                .until(this::assertJaegerHaveTrace, spanDataObj -> spanDataObj.get("data[0].spans[0].traceID") != null);
+        assertNotNull(spanData);
+        assertNotNull(spanData.get("data[0].spans[0].spanID"));
+        assertEquals("Test span", spanData.get("data[0].spans[0].operationName"));
+        assertEquals("Test event", spanData.get("data[0].spans[0].logs[0].fields[0].value"));
+        assertEquals("opentelemetry-integration-test-jeager-remote", spanData.get("data[0].processes.p1.serviceName"));
+        verifyResource(spanData);
+    }
+
+    private void createTestSpan(OpenTelemetry openTelemetry) {
+        Span span = openTelemetry.getTracer(getClass().getCanonicalName()).spanBuilder("Test span").startSpan();
+        try (Scope scope = span.makeCurrent()) {
+            span.addEvent("Test event");
+        } catch (Throwable t) {
+            span.recordException(t);
+            throw t;
+        } finally {
+            span.end();
+        }
+    }
+
+    private JsonPath assertJaegerHaveTrace() {
+        String serviceName = ConfigProvider.getConfig().getConfigValue("quarkus.application.name").getValue();
+        String path = String.format("/api/traces?service=%s", serviceName);
+
+        return given()
+                .baseUri(String.format(JAEGER_URL + ":%d", QUERY_PORT))
+                .contentType("application/json")
+                .when().get(path)
+                .body()
+                .jsonPath();
+    }
+
+    private void verifyResource(JsonPath spanData) {
```

Comment:
Sorry @hiteshkhatri97 but you should remove this method. None of these assertions is really relevant to understand if the jaeger sampler is actually being used. 
Let's assume it is for now and move on.

Reply:
updated